web/src/components/netflow-traffic.tsx

```diff
@@ -174,7 +174,9 @@ export const NetflowTraffic: React.FC<{
   const [match, setMatch] = React.useState<Match>(getMatchFromURL());
   const [reporter, setReporter] = React.useState<Reporter>(getReporterFromURL());
   const [layer, setLayer] = React.useState<Layer>(getLayerFromURL());
-  const [limit, setLimit] = React.useState<number>(getLimitFromURL());
+  const [limit, setLimit] = React.useState<number>(
```

Comment:
I think they should be two different settings, actually

Reply:
So you would keep both parameters in the URL and use only one ? :thinking:

Reply:
these are ultimately two different values for two different contexts, so yes I think it's better to decouple them, but that's ok for now we can live with it

Reply:
ok let's keep that in mind